processing/src/main/java/org/apache/druid/data/input/impl/DimensionSchema.java

```diff
@@ -176,6 +176,12 @@ public boolean hasBitmapIndex()
   @JsonIgnore
   public abstract ColumnType getColumnType();
 
+  @JsonIgnore
+  public boolean canBeMultiValued()
```

Comment:
Javadoc please. It always helps even for things that may be obvious to certain eyes.